### DISCUSSION TITLE
Ubuntu: impossible to install packages

### DISCUSSION THREAD

**Discussion by nico-tse226** [atom/atom#22213] [Q&A]
Hello, 
I am a beginner here, I have just installed Atom 1.56 on Ubuntu 20.04 (downloaded from atom website using the .deb file)

I try installing a package called language-stata, but cannot make it.
I work from home, so should not be using a proxy, and I already tried dealing with a firewall using
`apm config set strict-ssl false``

When I type apm config , I get this

`npm ERR! Usage:
npm ERR! npm config set <key> <value>
npm ERR! npm config get [<key>]
npm ERR! npm config delete <key>
npm ERR! npm config list [--json]
npm ERR! npm config edit
npm ERR! npm set <key> <value>
npm ERR! npm get [<key>]
npm ERR! 
npm ERR! alias: c
npm config failed: 1
`
Many thanks for your help (and patience :-) )

**Comment by sadick254** [comment]
Are you able to install Atom 1.55?

**Reply by nico-tse226** [reply]
Thanks for your reply.
I have installed Atom 1.55 as you suggested (using the .deb from Github website) but the problem is still there. 
I also tried installing Atom 1.56 with the snap and with the official package repository). 

Whenever I want to install a package, I alway get the ETIMEDOUT message. Using atom --safe or the terminal provide the same result, I have no more ideas ....

**Reply by sadick254** [reply]
I have Atom 1.56 install via snap on Ubuntu `20.04.2 LTS X86 X64` and I was able to install atom packages successfully without any errors. I was not able to reproduce the error posted earlier.

**Reply by nico-tse226** [reply]
Thanks, I agree, it is working. I have changed my internet connexion and get it working. No idea why with my home internet connection, it was not working, Thanks for your help

**Reply by sadick254** [reply]
Am happy to know that you are able to install packages now. Happy Coding 🚀
